### PR TITLE
Update nc_chi_squared.qbk

### PR DESCRIPTION
**Pull request by ambreshbiradar9**

### CODE REVIEW

File: doc/distributions/nc_chi_squared.qbk [hunk 1]
```diff
@@ -20,7 +20,7 @@
       // Constructor:
       non_central_chi_squared_distribution(RealType v, RealType lambda);
 
-      // Accessor to degrees of freedom parameter v:
+      // Accessor to degrees of freedom parameter [nu]:
```
Comment (NAThompson) [inline doc/distributions/nc_chi_squared.qbk:23]: Does [nu] get rendered correctly in the code bracket?

**Comment by NAThompson** [comment]
@ambreshbiradar9 : Why did you mark this as a draft? It looks like everything is ready go now.

**Comment by ambreshbiradar9** [comment]
> @ambreshbiradar9 : Why did you mark this as a draft? It looks like everything is ready go now.

ya it is ready 
sorry to convert to draft

**Comment by NAThompson** [comment]
Click the "Ready for review" button once it's ready to go.

**Comment by NAThompson** [comment]
Good job; looks like you're getting a bit more familiar with the workflow here.

**Comment by NAThompson** [comment]
@ambreshbiradar9 : If you're looking for a problem with a bit more meat, I broke the build for the quadrature routines because I used transitive header includes. So the correct header needs to be found and added to the quadrature routines so they compile on windows.

**Comment by ambreshbiradar9** [comment]
> @ambreshbiradar9 : If you're looking for a problem with a bit more meat, I broke the build for the quadrature routines because I used transitive header includes. So the correct header needs to be found and added to the quadrature routines so they compile on windows.

Can you provide me the link @NAThompson 

Also,
 I am having problems in building b2 engine @NAThompson I will ask you specifics if I need help

**Comment by NAThompson** [comment]
@ambreshbiradar9 : [Here's](https://ci.appveyor.com/project/jzmaddock/math/builds/33507129/job/kdciwnoq6huw7ssq) the build log.

**Comment by ambreshbiradar9** [comment]
hey @NAThompson sorry for asking this 
I am a noob 
for setting up i  have clonned boost/math but what is b2 i am having problems 
image

**Comment by NAThompson** [comment]
@ambreshbiradar9 : 

```
boost$ ./bootstrap.sh # should create b2
boost$ cd libs/math/test/
boost/libs/math/test$ ../../../b2
```

**Comment by ambreshbiradar9** [comment]
Hey, @NAThompson can you link me to the problem regarding quadrature routines

**Comment by NAThompson** [comment]
@ambreshbiradar9 : See line 961 of [this](https://travis-ci.org/github/boostorg/math/jobs/698229545) build log.

**Pull request merged**